sdk-tools/src/edger8r.rs

```diff
@@ -383,4 +429,65 @@ mod tests {
             assert!(!file.exists())
         }
     }
+
+    #[test]
+    fn dependent_edl_in_search_paths() {
```

Comment:
Up for suggestions on how to make this more consise.  Was on the fence if functionalizing more would be clearer or just another level of indirection for the test

Reply:
Long names aren't as much of a problem for tests, specifically, I think. That may just be a stylistic opinion of mine, though.

Reply:
Sorry, I knew what I was thinking but failed to fully communicate.
I think the body of the test function is larger than I would like.
As for naming test functions, I agree that tests have more leniency here, and I need to work on improving naming tests I write.